### PR TITLE
NN-4002 bug fixes

### PR DESCRIPTION
**Pull request by timoj58**

### CODE REVIEW

File: server/views/partials/reviewPanelRadios.njk
Comment (sp-watson) [inline server/views/partials/reviewPanelRadios.njk:30]: This formatting is not good - we really have to review this formatter you are using.

For one, why is it using tabs? Also, is this a good way of formatting - to have newlines for single attributes?

File: integration_tests/integration/prisonerReportReview.spec.ts
Comment (sp-watson) [inline integration_tests/integration/prisonerReportReview.spec.ts:362]: Let's just hold this off for a minute... we need to check this with the designers as well

File: server/routes/prisonerReport/prisonerReportReviewValidation.ts
Comment (sp-watson) [inline server/routes/prisonerReport/prisonerReportReviewValidation.ts:59]: Just noticed these 4 lines are virtually the same as above. 

Personally I don't like this much duplication - but it isn't a big issue

File: server/routes/prisonerReport/prisonerReportReviewValidation.ts
Comment (sp-watson) [inline server/routes/prisonerReport/prisonerReportReviewValidation.ts:54]: I would prefer > 0 - just a bit clearer for me. But it's your call

File: integration_tests/integration/prisonerReportReview.spec.ts
Comment (sp-watson) [reply integration_tests/integration/prisonerReportReview.spec.ts:362]: Approved!

File: server/views/partials/reviewPanelRadios.njk
Comment (timoj58) [reply server/views/partials/reviewPanelRadios.njk:30]: that formatting is github, its fine in the code

**Review by sp-watson — APPROVED** [review]

**Pull request merged**
